Cybersecurity New Topic: Virus (#1701)
Codecademy/docs: content/cybersecurity/concepts/virus/virus.md

Description: 'A virus is a type of computer program that can infect a system with malicious code and replicate itself.'
Subjects:
  - 'Cybersecurity'
  - 'Computer Science'
Tags:
  - 'Cybersecurity'
  - 'Computer Science'
CatalogContent:
  - 'paths/cyber-resilience-risk-management'
  - 'paths/fundamentals-of-cybersecurity'
---

A **virus** is a type of computer program that can infect a system with malicious code and replicate itself. Computer viruses have two main functions:

1. Destruction refers to the damage a computer virus can cause to a computer by deleting files or reformatting the hard drive.
2. Propagation is when a virus self-replicates by placing a copy of itself into other programs across a network.

## Virus In-Use

A virus's payload delivers the destructive power and negatively impacts the confidentiality, integrity, or availability of systems or data. Common signs of computer viruses can include unwanted pop-up windows, programs that self-execute, accounts being logged out, and crashing of the device. The risk of a virus can be mitigated by installing anti-virus software, scanning on a regular basis, and backing up data.

## Different Types of Viruses

> - Malware: Short for malicious software, malware is any type of software that could harm a computer system. It can make a computer perform actions without the owner’s consent or knowledge.

Reviewer: A virus is a type of malware, not the other way around

Reviewer: @SSwiniarski I think we should make a differentiation between viruses, worms, and malware (the umbrella term).

@jonchery In this section on different types of viruses, I'm wondering if we can cover the different types of viruses like File infectors, Boot sector viruses, Polymorphic viruses, and Stealth viruses.

- Trojan Horse: A type of malware that uses code to install seemingly benign software, but it is actually used to create back doors into a system. The result is typically loss or theft of data.

> - Worm: A piece of malicious software that replicates itself in order to spread to other systems. It often traverses across a network, exploiting security flaws on the target system to allow access.

Reviewer: Worms and viruses differ in how they replicate. Virus's attach themselves to programs or executable files while works can travel through networks and can act independently.

Reviewer: Are trojans able to replicate? I don't believe so but would like to discuss on this further. I think a section can be made on other types of malware and a future entry can be made separately on these other types of malware.

---
Title: 'Virus'

> Description: 'A virus is a type of computer program that can replicate and infect a system with malicious code.'

Reviewer: ```suggestion
Description: 'A virus is a type of computer program that can infect a system with malicious code and replicate itself.'
```

Subjects:
  - 'Cybersecurity'
  - 'Computer Science'
Tags:
  - 'Cybersecurity'
  - 'Computer Science'
CatalogContent:
  - 'paths/cyber-resilience-risk-management'
  - 'paths/fundamentals-of-cybersecurity'
---

> A **virus** is a type of computer program that can replicate and infect a system with malicious code. Computer viruses have two main functions: propagation and destruction. Propagation is when a virus self-replicates by placing a copy of itself into other programs across a network. Destruction refers to the damage a computer virus can cause to your computer by deleting files or reformatting your hard drive.

Reviewer: ```suggestion
A **virus** is a type of computer program that can infect a system with malicious code and replicate itself. Computer viruses have two main functions:

1. Destruction refers to the damage a computer virus can cause to a computer by deleting files or reformatting the hard drive.
2. Propagation is when a virus self-replicates by placing a copy of itself into other programs across a network.
```

What happens as a result of it copying itself into other programs

## Virus In-Use

A virus's payload delivers the destructive power and negatively impacts the confidentiality, integrity, or availability of systems or data. Common signs of computer viruses can include unwanted pop-up windows, programs that self-execute, accounts being logged out, and crashing of the device. The risk of a virus can be mitigated by installing anti-virus software, scanning on a regular basis, and backing up data.

## Different Types of Viruses

> - Malware: short for malicious software. Malware is any type of software that could harm a computer system. Malware can make a computer perform actions without the owner’s consent or knowledge.
> - Trojan Horse: a type of malware that uses code to install seemingly benign software, but it is actually used to create back doors into a system. The result is typically loss or theft of data.
> - Worm: a piece of malicious software that replicates itself in order to spread to other systems. It often traverses across a network, exploiting security flaws on the target system to allow access.
> - Spyware: software that tries to monitor and gather information about a user without their knowledge or consent. Spyware can monitor and log activity on a target system, or gather credit card and other personally identifiable information.

Reviewer: ```suggestion
- Malware: Short for malicious software, malware is any type of software that could harm a computer system. It can make a computer perform actions without the owner’s consent or knowledge.
- Trojan Horse: A type of malware that uses code to install seemingly benign software, but it is actually used to create back doors into a system. The result is typically loss or theft of data.
- Worm: A piece of malicious software that replicates itself in order to spread to other systems. It often traverses across a network, exploiting security flaws on the target system to allow access.
- Spyware: Software that tries to monitor and gather information about a user without their knowledge or consent. It can monitor and log activity on a target system, or gather credit card and other personally identifiable information.
```
```suggestion
- Malware: short for malicious software. Malware is any type of software that could harm a computer system. Malware can make a computer perform actions without the owner’s consent or knowledge.
- Trojan Horse: a type of malware that uses code to install seemingly benign software, but it is actually used to create back doors into a system. The result is typically loss or theft of data.
- Worm: a piece of malicious software that replicates itself in order to spread to other systems. It often traverses across a network, exploiting security flaws on the target system to allow access.
- Spyware: software that tries to monitor and gather information about a user without their knowledge or consent. Spyware can monitor and log activity on a target system, or gather credit card and other personally identifiable information.
```

> 
> 
> 
> 
> 

Reviewer: ```suggestion

```

---
Title: 'Virus'

> Description: 'Virus is a type of computer program that can replicate and infect a system with malicious code.'

Reviewer 2: ```suggestion
Description: 'A virus is a type of computer program that can replicate and infect a system with malicious code.'
```

Subjects:
  - 'Cybersecurity'
  - 'Computer Science'
Tags:
  - 'Cybersecurity'
  - 'Computer Science'
CatalogContent:
  - 'paths/cyber-resilience-risk-management'
  - 'paths/fundamentals-of-cybersecurity'

>   - 'introduction-to-personal-digital-security'

Reviewer 2: ```suggestion
```
Generally only have two slugs under CatalogContent.

---

A **virus** is a type of computer program that can replicate and infect a system with malicious code. Computer viruses have two main functions: propagation and destruction. Propagation is when a virus self-replicates by placing a copy of itself into other programs across a network. Destruction refers to the damage a computer virus can cause to your computer by deleting files or reformatting your hard drive.

## Virus In-Use:

A virus's payload delivers the destructive power and negatively impacts the confidentiality, integrity, or availability of systems or data. Common signs of computer viruses can include unwanted pop-up windows, programs that self-execute, accounts being logged out, and crashing of the device. The risk of a virus can be mitigated by installing anti-virus software, scanning on a regular basis, and backing up data.

## Different Types of Viruses

> - Malware: short for malicious software. Malware is any type of software that could harm a computer system. Malware can make a computer perform actions without the owner’s consent or knowledge.

Reviewer 2: Remove line breaks between bullet points.
